crypto.go

```diff
@@ -57,6 +46,25 @@ func NewIdentity(privKey ic.PrivKey) (*Identity, error) {
 	}, nil
 }
 
+func newIdentity(sk ic.PrivKey) (identity, error) {
+	return NewIdentity(sk)
+}
```

Comment:
Why is this additional constructor needed?

Reply:
It's not. Removed